flexmeasures/ui/templates/sensors/status.html

```diff
@@ -1,363 +1,306 @@
-            </div>
+Sensors shown on this page are the ones relevant for correct functioning of the asset. We list the ones linked in the flex-context and the graphs page.
+          "></span>
+        </h4>
+
+        <div class="table-responsive">
+          <table id="sensorStatusTable" class="table table-striped paginate w-100"></table>
         </div>
+
+        <!-- JOBS TABLE -->
+        <h4 class="">
+          Latest jobs of {{ asset.name }}
+          <span class="fa fa-info-circle " title="This table shows forecasting or scheduling jobs for this asset."></span>
+          <span class="jobs-time-ago" data-timestamp="Loading">
+            <small>
+              <i class="fa fa-refresh" id="refresh_jobs" title="Refresh status"></i>
+              <span id="jobs_time_ago"></span>
+            </small>
+          </span>
+        </h4>
+
+        <div class="table-responsive">
+          <table id="jobsTable" class="table table-striped paginate nav-on-click w-100"></table>
+        </div>
+
+        <div class="alert alert-warning d-none mt-2" id="redis_connection_err"></div>
+      </div>
     </div>
+  </div>
 </div>
 
-<!-- hide control elements -->
 <script>
-    $(document).ready(function() {
-        $('#sensor_statuses').DataTable({
-            "searching": false,
-            "paging": false,
-            "info": false,
-            "ordering": false
-        });
-
-        if (sensors.length != 0) {
-            // Append a loading row to the sensor status table
-            $('#sensor_statuses tbody').append(getLoadingRow());
+  const assetId = "{{ asset.id }}";
+  const sensors = {{ sensors | tojson }};
+
+  function SensorStatusRow(sensor) {
+    const lastCallTime = Date.now();
+    const timeAgo = getTimeAgo(lastCallTime);
+    const statusIcon = sensor.stale ? "🔴" : "🟢";
+    const refreshIcon = `<i class="fa fa-refresh sensor_refresh" id="sensor_refresh_${sensor.id}"></i>`;
+
+    return {
+      sensor_info: `${sensor.name} (<a href="/sensors/${sensor.id}">${sensor.id}</a>)
+        <span class="fa fa-info" title="Resolution: ${sensor.resolution}, Asset: '${sensor.asset_name}', Reason: ${sensor.relation}" style="margin-left: 10px;"></span>`,
+      source_type: sensor.source_type || "None",
+      staleness_since: getHumanFriendlyDeltaOrTimeStr(sensor.staleness_since) || "Never",
+      status: `${statusIcon}<br><span class="time-ago" data-timestamp="${lastCallTime}">
+          <small>${refreshIcon} <span class="time-text">(${timeAgo})</span></small></span>`,
+      url: `/sensors/${sensor.id}`
+    };
+  }
+
+  function JobRow(job) {
+    const jobStatusMessage = job.err ? `${job.status}: ${job.err}` : job.status;
+    const jobStatusIcon = job.status === "finished" ? "🟢" :
+                          job.status === "failed" ? "🔴" : "🟡";
+    return {
+      created_at_timestamp: job.enqueued_at,
+      created_at: getHumanFriendlyDeltaOrTimeStr(job.enqueued_at),
```

Comment:
Better also secure this against the case of `job.enqueued_at` being `null`. Less likely than in the sensor data case, but better safe than sorry. If it is null, display "Unknown".